### DISCUSSION TITLE
Mail sending limit

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14795] [How to]
Gents, where I can set sending limit (per second) so when I do mass mail campaign, YF will send by batches, holding whole list in queue (resuming every 1 or 2 seconds? Appreciate help

**Comment by rskrzypczak** [comment]
You only have this at your disposal
https://github.com/YetiForceCompany/YetiForceCRM/blob/d03b240a5fcfb3fb5e8bf470cfc0b39ec8a60b0b/config/Performance.php#L80-L81

**Reply by waran70** [reply]
In  old phpmailer we could configure it per second/m  ==> Use sleep(1)
Indeed sending email in packets configurable per second or minute would help maintaining various smtp requirements.
For instance I have one EC with limit of 20 mails per second. It would be good thing for marketing (between other) related app... 
Thumb up to thread

**Comment by mariuszkrzaczkowski** [comment]
my suggestion:
1. run `ron/cron.sh` every 1 min
2. Cron task "Emails sender" (Software configuration > Automation > CRON) every 1 min
3. $CRON_MAX_NUMBERS_SENDING_MAILS = 1000;   

you will get the effect of 1000 e-mails per minute
